src/main/java/duke/io/Storage.java

```diff
@@ -0,0 +1,141 @@
+package duke.io;
+
+import duke.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * Storage class define rules for create, read and write to file.
+ *
+ * @author Galvin Leow Wen Yuan
+ * @author A0200204J
+ * @version v1.0
+ */
+public class Storage {
+    protected String filePath;
+
+    /**
+     * Class constructor.
+     * Initialise Storage class with filepath.
+     *
+     * @param filePath use to create, read and write a file.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Read file and convert it into Arraylist<Task>.
+     *
+     * @return Arraylist<Task> contains all the task from the file.
+     * @throws DukeException exception for file cannot be found.
+     */
+    public ArrayList<Task> load() throws DukeException {
+        ArrayList<Task> taskArrayList = new ArrayList<>();
+        File file = new File(filePath);
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(file));
+            String fileLine;
+            while ((fileLine = br.readLine()) != null) {
+                String[] tempArr = fileLine.split(",");
+                String command = tempArr[0];
+                switch (command) {
+                    case "duke.task.Todo":
+                        taskArrayList.add(new Todo(tempArr[2]));
+                        break;
+                    case "duke.task.Deadline":
+                        Task tempDeadline = new Deadline(tempArr[2], LocalDateTime.parse(tempArr[3]));
+                        if (tempArr[1].equals("true")) {
+                            tempDeadline.markAsDone();
+                        }
+                        taskArrayList.add(tempDeadline);
+                        break;
+                    case "duke.task.Event":
+                        Task
```

Comment:
Should there be a line break here between line 67 and 68? I feel that there is no need for a line break as the line is not long when put together.